Cargo.toml

```diff
@@ -4,7 +4,7 @@ name    = "sdl2"
 description = "SDL2 bindings for Rust"
 repository = "https://github.com/AngryLawyer/rust-sdl2"
 documentation = "http://angrylawyer.github.io/rust-sdl2/sdl2/"
-version = "0.30.0"
+version = "0.30.1"
```

Comment:
Well, I modified Cargo.toml so it seemed reasonable to make it a micro version bump since no code was changed. Also, I just tended to prefer to not develop with a version that matches a published version.